src/util/requestDispatcher.js

```diff
@@ -27,15 +30,26 @@ export default class RequestDispatcher {
      * and the tries are less than max tries
      * keep retrying
      */
-    if (!isValueValid(this.device.gcookie) &&
+
+    if (evtName && evtName === 'wzrk_fetch') {
```

Comment:
ideally we should not have these strings in our code. We can move this `wzrk_fetch` to a constants file and reuse it

Reply:
Fixed this.